FrameworkSystem/private/standardLogging/gLogging.py

```diff
@@ -0,0 +1,335 @@
+"""
+gLogging
+"""
+
+__RCSID__ = "$Id$"
+
+import logging
+
+from DIRAC.FrameworkSystem.private.standardLogging.LogLevels import LogLevels
+
+from DIRAC.FrameworkSystem.private.standardLogging.Backend.StdoutBackend import StdoutBackend
+from DIRAC.FrameworkSystem.private.standardLogging.Backend.StderrBackend import StderrBackend
+from DIRAC.FrameworkSystem.private.standardLogging.Backend.FileBackend import FileBackend
+from DIRAC.FrameworkSystem.private.standardLogging.Backend.RemoteBackend import RemoteBackend
+
+
+class gLogging(object):
+  """
+  gLogging is a wrapper of the logger object from the standard logging library which integrate
+  some DIRAC concepts. It is the equivalent to the old gLogger object.
+
+  It is used like an interface to use the logger object of the logging library. 
+  Its purpose is to replace transparently the old gLogger object in the existing code in order to 
+  minimize the changes. 
+
+  In this way, each gLogging embed a logger of logging. It is possible to create sublogger,
+  set and get the level of the embedded logger and create log messages with it.
+
+  gLogging could delegate the initialization and the configuration to a factory of the root logger be it can not
+  because it has to wrap the old gLogger.  
+  """
+
+  _componentName = "Framework"
```

Comment:
I added a comment on its presence. 
It is the default name for the system/component name that you can find in the log messages. Do you understand ?